Docs/details/santabs.md

```diff
@@ -0,0 +1,44 @@
+# santabs
+
+The santabs process is an XPC service for the santa-driver.kext bundle, meaning only binaries within that bundle can launch santabs. It will be launched with with whatever privileges the calling process is running as. Currently santad is the only caller of santabs, so santabs runs as root.
+
+#####Events
+
+santabs is quite simple and only does one thing. It generates non-execution events for the contents of a bundle. 
```

Comment:
Santabs is quite simple and only does one thing: it generates non-execution events for the contents of a bundle.